Form/Editors: describe `isDirty`, `clear` and `reset` (#5579)
DevExpress/devextreme-documentation: api-reference/10 UI Components/Editor/1 Configuration/isDirty.md

---
---
##### shortDescription
Specifies whether the component's current value differs from the initial value.

##### readonly

---

> This property is a readonly flag. You can use it to check whether the editor was changed.

Reviewer: I would be a bit more pedantic here and use a hyphen for "read-only". I'm sure some resources may now write it as one word, but it just seems wrong to me :-) 

"editor was changed" is not an entirely correct phrase. If you resize an editor, or change its color, you're changing the editor. Let's incorporate the word "value" here to be more specific. I'm also not sure that "changed" or "was changed" differ in this context. 

```suggestion
This property is a read-only flag. You can use it to check whether the editor value changed.
```

> This property is a read-only flag. You can use it to check whether the editor value changed.

Reviewer 2: ```suggestion
This property is a read-only flag. You can use it to check if the editor value changed.
```
